lib/SILOptimizer/Mandatory/TFPartition.cpp

```diff
@@ -115,11 +115,11 @@ enum class PartitioningClass {
 
   /// This is an explicit send to the accelerator, which is sugared as
   /// '.toAccelerator()'.
-  ExplicitSend,
+  ExplicitToAccel,
 
   /// This is an explicit receive from the accelerator, which is sugared as
   /// '.toHost()'.
-  ExplicitReceive,
+  ExplicitToHost,
```

Comment:
Please correct me if I am wrong. ExplicitToHost would result in a send in the accelerator code and a receive in the host code. Right?

Reply:
Yes